src/contracts/bodhi_token.js

```diff
@@ -54,12 +52,16 @@ const BodhiToken = {
       return;
     }
 
-    const oracle = new qweb3.Contract(Contracts.BodhiToken.address, Contracts.BodhiToken.abi);
-    return await oracle.call('balanceOf', {
+    const contract = getContract();
+    return await contract.call('balanceOf', {
       methodArgs: [owner],
       senderAddress: senderAddress,
     });
   },
 };
 
+function getContract() {
+  return new Contract(Config.QTUM_RPC_ADDRESS, Contracts.BodhiToken.address, Contracts.BodhiToken.abi);
+}
+
```

Comment:
This Contract instance can be a constant on top of this page.

Reply:
good call, fixed